primo/opt_model/tests/test_model_options.py

```diff
@@ -154,7 +152,7 @@ def test_opt_model_inputs(get_column_names):
     assert "Clusters" in wd_gas
 
     opt_mdl_inputs.build_optimization_model()
-    opt_campaign = opt_mdl_inputs.solve_model(solver="scip")
+    opt_campaign = opt_mdl_inputs.solve_model(solver="highs")
     opt_mdl = opt_mdl_inputs.optimization_model
```

Comment:
This reminds me we need to change the solver option for the API

Reply:
Thanks Will!